flasc/energy_ratio/energy_ratio_visualization.py

```diff
@@ -20,7 +20,7 @@
 from floris import tools as wfct
 
 
-def plot(energy_ratios, labels=None):
+def plot(energy_ratios, labels=None, label_uq=False):
```

Comment:
I think the variable `label_uq` can be confusing. Users may try to insert a list with the labels for the uncertainty bounds. Perhaps we can change it to `hide_uq_labels=True`?